ltr/sagemaker/learntorank.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM public.ecr.aws/docker/library/python:3.10
```

Comment:
Probably worth trying the `-slim` or `-alpine` version for a quick win here (mainly in terms of reducing the number of vulnerability reports for packages that we're probably not using!)

Reply:
Good shout - updated to use the slim version. Alpine is missing some system dependencies needed for pandas.